### PR TITLE
Ported oeedger8r to Windows.

### PR DESCRIPTION
**Pull request by gupta-ak**

### CODE REVIEW

File: docs/GettingStartedDocs/GettingStarted.Windows.md [hunk 1]
```diff
@@ -86,6 +87,14 @@ $ cmake ..
 $ make
 ```
 
+### Configuring OCaml for Windows
+Before building, there are a few OCaml configuration steps:
+
+1. Add `ocaml-env.exe` to the `PATH` environment variable. The default path of `ocaml-env.exe` is  `C:\OCaml64\usr\local\bin\ocaml-env.exe`.
+2. Open `cmd.exe` (or another shell) and run `ocaml-env exec -- cmd.exe`. This is create a OCaml environment in the shell.
```
Comment (anakrish) [inline docs/GettingStartedDocs/GettingStarted.Windows.md]: This creates

File: docs/GettingStartedDocs/GettingStarted.Windows.md [hunk 2]
```diff
@@ -86,6 +87,14 @@ $ cmake ..
...
+3. Install `ocamlbuild` by running `opam install ocamlbuild`.
+4. Exit out of the shell.
```
Comment (anakrish) [inline docs/GettingStartedDocs/GettingStarted.Windows.md]: Is this a one time thing or every time you build?

File: tools/CMakeLists.txt [hunk 3]
```diff
@@ -2,5 +2,8 @@
 # Licensed under the MIT License.
 
 add_subdirectory(oeedger8r)
+
+if (UNIX)
 add_subdirectory(oesgx)
 add_subdirectory(oesign)
+endif()
```
Comment (anakrish) [inline tools/CMakeLists.txt]: Needs newline.

File: docs/GettingStartedDocs/GettingStarted.Windows.md (hunk 2 above)
Comment (gupta-ak) [reply docs/GettingStartedDocs/GettingStarted.Windows.md]: Yes, it's a one time thing. I'll make that more clear in the docs.

**Comment by gupta-ak** [comment]
@anakrish I've updated the PR based off your feedback. If it looks good to you, could you approve?

**Comment by paulcallen** [comment]
bors retry

**Comment by gupta-ak** [comment]
bors retry

**Pull request merged**